Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py

```diff
@@ -3969,7 +3969,447 @@ def handle_message(item: results.Message | dict) -> bool:
         return True
     return False
 
+def splunk_create_investigation(
+    base_url,
+    name,
+    description=None,
+    investigation_type=None,
+    status=None,
+    disposition=None,
+    owner=None,
+    urgency=None,
+    sensitivity=None,
+    finding_ids=None,
+    finding_times=None,
+    auth_token=None,
+    sessionKey=None,
+    verify=VERIFY_CERTIFICATE
+):
+    if not sessionKey and not auth_token:
+        return_error("A session_key/auth_token was not provided")
+
+    headers = {
+        "Authorization": f"Bearer {auth_token}" if auth_token else sessionKey,
+        "Content-Type": "application/json"
+    }
+
+    url = f"{base_url}servicesNS/nobody/missioncontrol/public/v2/investigations"
+
+    payload = {
+        "name": name,
+        "description": description,
+        "investigation_type": investigation_type,
+        "status": status,
+        "disposition": disposition,
+        "owner": owner,
+        "urgency": urgency,
+        "sensitivity": sensitivity,
+        "finding_ids": finding_ids,
+        "finding_times": finding_times
+    }
+    payload = {k: v for k, v in payload.items() if v is not None}
+
+
+    response = requests.post(url, headers=headers, json=payload, verify=verify)
+
+    if response.status_code not in (200, 201):
+        return_error(f"[Investigation] Creation failed. Status: {response.status_code}, Body: {response.text}")
+
+    return response.json()
+
+def splunk_create_investigation_command(base_url: str, token: str, auth_token: str | None, args: dict) -> None:
+    session_key = None if auth_token else token
+
```

Comment:
why don't you check here as well if one of the session_key or auth_token does exist instead of going into the helper function and then checking?